app/src/androidTest/java/ch/epfl/sdp/drone3d/map/area/ParallelogramAreaBuilderTest.kt

```diff
@@ -56,4 +57,25 @@ class ParallelAreaBuilderTest {
         area.reset()
         assertThat(area.vertices.size, equalTo(0))
     }
+
+    @Test
+    fun getAreaSizeIsCorrect() {
+        val area = ParallelogramBuilder()
+
+        val a = LatLng(Random.nextDouble(), Random.nextDouble())
+        val b = LatLng(Random.nextDouble(), Random.nextDouble())
+        val c = LatLng(Random.nextDouble(), Random.nextDouble())
```

Comment:
Random.nextDouble() gives values between 0 and 1 and I understand that it is to not create a too big area but it causes to only test within this small range. To diversify the testing, we could generate a random latitude and longitude from which we will create the area. Maybe looking at my code suggestion gives you an idea of what I'm thinking.
```suggestion
        val initialLat = Random.nextDouble(-90.0, 89.0)
        val initialLng = Random.nextDouble(-180.0, 179.0)
        val a = LatLng(initialLat + Random.nextDouble(), initialLng + Random.nextDouble())
        val b = LatLng(initialLat + Random.nextDouble(), initialLng + Random.nextDouble())
        val c = LatLng(initialLat + Random.nextDouble(), initialLng + Random.nextDouble())
```

Reply:
You're right thanks